packages/cxl-ui/src/index.js

```diff
@@ -7,6 +7,7 @@ export { CXLSectionElement } from './components/cxl-section.js';
 
 // Order matters.
 export { CXLVaadinAccordion } from './components/cxl-vaadin-accordion.js';
+export { CxlPlaybookAccordion } from './components/cxl-playbook-accordion.js';
```

Comment:
Missing caps `CXL` in class name.

Reply:
fixed in https://github.com/conversionxl/aybolit/pull/87/commits/ad17fceadb9eb7e63751becfe07ba9a8de400a7c